Sming/Libraries/SDCard/SDCard.cpp

```diff
@@ -38,19 +38,29 @@ Descr: Low-level SDCard functions
 FATFS *pFatFs = NULL;		/* FatFs work area needed for each volume */
 SPIBase *SDCardSPI = NULL;
 uint8 SPI_CS;				/* SPI client selector */
+uint8 SPI_byte_order;		/* The order the SPI bytes are sent */
+uint32 SPI_freq_limit;		/* The max frequency the SPI should run */
+uint32 SPI_init_freq;		/* SPI frequency usined for init */
 
 
 #define SCK_SLOW_INIT 10
 #define SCK_NORMAL 0
 
-void SDCard_begin(uint8 PIN_CARD_SS)
+void SDCard_begin(uint8 PIN_CARD_SS, uint8 byte_order, uint32 freq_limit)
```

Comment:
Please, use camelCase for the variable names. For example `byteOrder` instead of `byte_order`. 
Once you are ready with the changes make sure to run the coding style checker: https://sming.readthedocs.io/en/latest/contribute/coding-style.html

Reply:
changed to camelCase and run through clang